test/pgtest-mz/datums.pt

```diff
@@ -54,10 +54,10 @@ DataRow {"fields":["\u0000\u0000\u0000\u0000\u0000\u0000\u0000\t","NULL","NULL",
 DataRow {"fields":["\u0000\u0000\u0000\u0000\u0000\u0000\u0000\n","NULL","NULL","NULL","NULL","NULL","NULL","NULL","[255, 128, 0, 0]","[255, 240, 0, 0, 0, 0, 0, 0]","[0, 0, 255, 255, 240, 0, 0, 0]","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","\u0001-1","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL"]}
 DataRow {"fields":["\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0007","NULL","NULL","NULL","NULL","NULL","NULL","NULL","4\u0000\u0000\u0000","[60, 176, 0, 0, 0, 0, 0, 0]","[0, 9, 0, 0, 0, 0, 0, 31, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 8, 156, 17, 252, 36, 34, 12, 58]","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","\u0001\"\\\"\"","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL"]}
 DataRow {"fields":["\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0006","NULL","[127, 255]","[127, 255, 255, 255]","[127, 255, 255, 255, 255, 255, 255, 255]","NULL","NULL","NULL","[127, 127, 255, 255]","[127, 239, 255, 255, 255, 255, 255, 255]","[0, 0, 255, 255, 208, 0, 0, 0]","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","NULL","\u0001\"'\"","NULL","NULL","NULL","NULL","NULL","NULL","NULL","p\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000u*\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u000f\u0003\u0000\u0000\u0000\u0000\u0000\u0000"]}
-DataRow {"fields":["\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0005","NULL","[128, 1]","[128, 0, 0, 1]","[128, 0, 0, 0, 0, 0, 0, 1]","NULL","NULL","NULL","[0, 128, 0, 0]","\u0000\u0010\u0000\u0000\u0000\u0000\u0000\u0000","[0, 0, 255, 255, 0, 0, 0, 0]","NULL","NULL","NULL","NULL","NULL","NULL","NULL",".","NULL",".","\u0001\" \"","NULL","NULL","NULL","NULL","NULL","NULL","NULL","p\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000u*\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"]}
+DataRow {"fields":["\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0005","NULL","[128, 1]","[128, 0, 0, 1]","[128, 0, 0, 0, 0, 0, 0, 1]","NULL","NULL","NULL","[0, 128, 0, 0]","\u0000\u0010\u0000\u0000\u0000\u0000\u0000\u0000","\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000","NULL","NULL","NULL","NULL","NULL","NULL","NULL",".","NULL",".","\u0001\" \"","NULL","NULL","NULL","NULL","NULL","NULL","NULL","p\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000u*\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"]}
```

Comment:
Ergh. Real unfortunate we had this file under pgtest-mz, because this would have been caught if it were run against postgres, right?

Reply:
Yep